README.md

```diff
@@ -1,15 +1,142 @@
+
+- Models:We are using 3 models named "count.model,named-id-required.model,named-id.model" In these models we are using model based validations.
+
+## Auth:
+
+- Adaptors:Two adapter service provided are "logged-in-user-adapter.service and login-adapter.
+  service" which helps in adapting API response to a LoggedInUserDM model representing a logged-in user or adapting the LoginModel object to the API request format and adapting the API response which allows for a separation of concerns between the API response data and the application data model.
+
+- Commands:It provides implementation of a commands that makes a Get/Post/patch request to the
+  API endpoint for triggering forget password functionality,retrieving current user's data,authentication token,initiating a Google OAuth login flow and logout etc.
+
+- Guards:In microservice 2 guards provided are "LoggedInGuard and AuthGuard" used to control
+  access to certain routes or pages in the application.Guards can be used to check if the user has permission to access a certain route, if the user is logged in, or to perform other types of authentication or authorization check.
+
+- Models: We are using 4 models named "logged-in-user.model,loggedin-response.model,login.model,
+  token-response.model" that represents user's data when they are logged in,response from a login API call(code),data that is required for a user to authenticate or log
+  in to a system and the the response from server after a user gets authenticated
+  and obtains a new access token and refresh token.
+
+## Decorator:
+
+In this microservice we are using 2 decorators named "required.decorator,validate.decorator",In these decorator we are providing our requirement for the data that which data needs to be fill,and than validate the data.
+
+## Interceptors:
+
+In this microservice we are using 3 Interceptors named auth.interceptor,error.interceptor,session-recovery.interceptor.So, in these we are providing header as token & checking when that gives us the error and also checking when will session get expired.
+
+## Localization:
+
+In this microservice we are using i18n module as a language translator & where we are using 3 files as follows :
+
+- Enums:we are using language.enum in this microservice where this files define the set of named values.
+
+- Module:We are using localization.module while using the module we are using data from enum too and according to enums value we translate the language and call the service
+
+- Service: The translationService provides localization functionality to the application. It depends on @ngx-translate/core library to handle translations allowing users to select their preferred language for the application & to provide appropriate translations based on that preference.
+
+## Env Resolver Service:
+
+The purpose of this service is to retrieve environment configuration data from a store (SystemStoreFacadeService) and make it available to components before they are displayed.
```

Comment:
Mention an example of how components can use env config resolved using this service.